include/vrv/floatingobject.h

```diff
@@ -265,6 +279,14 @@ class FloatingPositioner : public BoundingBox {
      */
     void CalcDrawingYRel(Doc *doc, const StaffAlignment *staffAlignment, const BoundingBox *horizOverlappingBBox);
 
+    /**
+     * Align extender elements across systems
+     */
+    void AlignExtenders();
```

Comment:
Since this is part of the "adjustments" steps in the page layout, maybe rename it to "AdjustExtenders" ?